scripts/generate_contributors_list.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+
+ContributorsPath=../CONTRIBUTORS.txt
+
+echo 'For the purpose of tracking copyright, this is the list of individuals and
```

Comment:
probably works nicer as a 'here document':

cat > "$contibutors_path" <<EOF
For the purpose of tracking copyright, [...]

[...]

### Contributors
EOF

Reply:
snake_case is fine for me. It looks like the word "contributors" attracts typos haha. You spelled it twice without `r`